app/client/components/adminUserList.jsx

```diff
@@ -199,7 +199,7 @@ class AdminUserList extends React.Component {
       );
 	  }
 
-	  if (this.props.login.adminTier === 'view' || 'subadmin') {
+	  if (this.props.login.adminTier === 'view' || this.props.login.adminTier === 'subadmin') {
```

Comment:
@enjeyw double-checking subadmin/enroller type _shouldn't_ have access to the admin list?

This reflects the current auth on backend - `@requires_auth(allowed_roles={'ADMIN': 'admin'})`